config/environments/production.rb

```diff
@@ -34,37 +34,31 @@
                          [:memory_store, { size: 64.megabytes }]
                        end
 
-  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
```

Comment:
i dont know anything about our secrets settings, but guessing we still need this

Reply:
Nope. We're not using encrypted secrets at all.